pkg/ingester/ingester_v2.go

```diff
@@ -1620,11 +1661,16 @@ func (i *Ingester) closeAndDeleteUserTSDBIfIdle(userID string) tsdbIdleCheckResu
 
 	// And delete local data.
 	if err := os.RemoveAll(dir); err != nil {
-		level.Error(util.Logger).Log("msg", "failed to delete idle TSDB", "user", userID, "err", err)
+		level.Error(util.Logger).Log("msg", "failed to delete local TSDB", "user", userID, "err", err)
 		return tsdbDataRemovalFailed
 	}
 
-	level.Info(util.Logger).Log("msg", "deleted data for idle TSDB", "user", userID, "dir", dir)
+	if tenantDeleted {
+		level.Info(util.Logger).Log("msg", "deleted local TSDB, user marked for deletion", "user", userID, "dir")
```

Comment:
The log messages ends with `"dir"` but the dif value is missing.

```suggestion
		level.Info(util.Logger).Log("msg", "deleted local TSDB, user marked for deletion", "user", userID, "dir", dir)
```